esmvaltool/recipes/cmorizers/recipe_era5.yml

```diff
@@ -0,0 +1,269 @@
+        era5_freq: hourly
+        preprocessor: daily_mean
+      tas:
+        mip: E1hr
+        era5_name: 2m_temperature
+        era5_freq: hourly
+        preprocessor: daily_mean
+      evspsbl:
+        mip: E1hr
+        era5_name: evaporation
+        era5_freq: hourly
+        preprocessor: daily_mean
+      tasmax:
+        mip: E1hr
+        era5_name: maximum_2m_temperature_since_previous_post_processing
+        era5_freq: hourly
+        preprocessor: daily_max
+      psl:
+        mip: E1hr
+        era5_name: mean_sea_level_pressure
+        era5_freq: hourly
+        preprocessor: daily_mean
+      rls:
+        mip: E1hr
+        era5_name: mean_surface_net_long_wave_radiation_flux
+        era5_freq: hourly
+        preprocessor: daily_mean
+      tasmin:
+        mip: E1hr
+        era5_name: minimum_2m_temperature_since_previous_post_processing
+        era5_freq: hourly
+        preprocessor: daily_min
+      evspsblpot:
+        mip: E1hr
+        era5_name: potential_evaporation
+        era5_freq: hourly
+        preprocessor: daily_mean
+      ptype:
+        mip: E1hr
+        era5_name: precipitation_type
+        era5_freq: hourly
+        preprocessor: daily_mean # what to do? Mode?
+      mrro:
+        mip: E1hr
+        era5_name: runoff
+        era5_freq: hourly
+        preprocessor: daily_mean
+      ts:
+        mip: E1hr
+        era5_name: skin_temperature
+        era5_freq: hourly
+        preprocessor: daily_mean
+      prsn:
+        mip: E1hr
+        era5_name: snowfall
+        era5_freq: hourly
+        preprocessor: daily_mean
+      rss:
+        mip: E1hr
+        era5_name: surface_net_solar_radiation
+        era5_freq: hourly
+        preprocessor: daily_mean
+      ps:
+        mip: E1hr
+        era5_name: surface_pressure
+        era5_freq: hourly
+        preprocessor: daily_mean
+      rsds:
+        mip: E1hr
+        era5_name: surface_solar_radiation_downwards
+        era5_freq: hourly
+        preprocessor: daily_mean
+      rlds:
+        mip: E1hr
+        era5_name: surface_thermal_radiation_downwards
+        era5_freq: hourly
+        preprocessor: daily_mean
+      tsn:
+        mip: E1hr
+        era5_name: temperature_of_snow_layer
+        era5_freq: hourly
+        preprocessor: daily_mean
+      rsdt:
+        mip: E1hr
+        era5_name: toa_incident_solar_radiation
+        era5_freq: hourly
+        preprocessor: daily_mean
+      clt:
+        mip: E1hr
+        era5_name: total_cloud_cover
+        era5_freq: hourly
+        preprocessor: daily_mean
+      pr:
+        mip: E1hr
+        era5_name: total_precipitation
+        era5_freq: hourly
+        preprocessor: daily_mean
+
+  monthly:
+    description: CMORize monthly ERA5 data
+    scripts: null
+    variables:
+      pr:
+        mip: Amon
+      uas:
+        mip: Amon
+      vas:
+        mip: Amon
+      tas:
+        mip: Amon
+      tasmin:
+        mip: Amon
+      tasmax:
+        mip: Amon
+
```

Comment:
It would be great if we could have the same list of variables for all three frequencies. Would that be feasible? Or is not all data available?

Reply:
I think all data should be available, but I don't have a local copy for the monthly data. Just tested with 1 variable so far. I adjusted the recipe.